js/app.js

```diff
@@ -118,7 +118,7 @@ firebase.initializeApp(config);
             const u_tracks_Options = u_track.querySelectorAll('option');
 
             // User Current Info [ Top Summary ]
-            userPreferences.innerHTML = `<i class="fas fa-certificate"></i> ${doc.data().userTrack} <i class="fas fa-bug"></i> ${doc.data().currentProject} <br><i class="fas fa-globe"></i> ${doc.data().languageFirst}, ${doc.data().languageSecond}`;
+            userPreferences.innerHTML = `<i class="fas fa-certificate"></i> ${doc.data().userTrack} <i class="fas fa-bug"></i>${doc.data().currentProject} <br><i class="fas fa-globe"></i> ${doc.data().languageFirst}, ${doc.data().languageSecond}`;
```

Comment:
I wonder why do we display the `track` and `project` name. It is intentionally known. As the user will pick a `Track` at the beginning, then choose a `Project`. So, it is obvious that displayed users would be in the selected `Track` and `Project`.

What do you think of it?

Reply:
hmm the extra space here that I made was not intentional, sorry...  Regarding your question - is it not the part with our settings, which is on the top, next to the photo? I mean the one from the logged in user?